packages/onboarding/src/sensei-step-container/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { Icon, wordpress } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import React from 'react';
+import StepContainer from '../step-container';
+import { SENSEI_FLOW } from '../utils';
+import { Container, TitleContainer, Title, Footer, FooterText } from './components';
+import './styles.scss';
+
+interface SenseiStepContainerProps {
+	stepName: string;
+	recordTracksEvent: ( eventName: string, eventProperties: object ) => void;
+	children: React.ReactNode;
+}
+
+export const SenseiStepContainer: React.FC< SenseiStepContainerProps > = ( {
+	children,
+	...props
+} ) => {
+	const { __ } = useI18n();
+	return (
+		<StepContainer
+			flowName={ SENSEI_FLOW }
+			isWideLayout
+			hideFormattedHeader
+			shouldHideNavButtons
+			{ ...props }
+			stepContent={
+				<Container>
+					<TitleContainer>
+						<Icon icon={ wordpress } />
+						<Title>{ __( 'Course Creator' ) }</Title>
+					</TitleContainer>
+					{ children }
+					<Footer>
+						<FooterText>
+							{ createInterpolateElement( __( 'Hosting with the <a>WordPress.com</a>' ), {
```

Comment:
We've received a comment from a translator that `Hosting with the WordPress.com` sounds awkward. Should it be `Hosted by WordPress.com` or `Hosting with WordPress.com`?

Reply:
Good point fixed in a965d60e28bb449a8e097aa332967171e47e3256